tests/test_publish_grok46_hub.py

```
@@ -0,0 +1,290 @@
+                    {
+                        "factory": ITEM["slug"],
+                        "round": 1,
+                        "commit_point": marker.name,
+                        "files": [{"name": batch.name, "sha256": "0" * 64}],
+                    },
+                    "hash mismatch",
+                ),
+            ):
+                with self.subTest(message=message):
+                    marker.write_text(json.dumps(manifest) + "\n")
+                    with self.assertRaisesRegex(SystemExit, message):
```

Ruff (0.16.1)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)